ch9.md

```diff
@@ -311,14 +311,14 @@ If we want to keep the Observer and Observable separate:
 
 // producer:
 
-var a = Rx.Observable.create( function onObserver(observer){
+var a = Rx.Observable.create( function onObserve(observer){
 	setInterval( function everySecond(){
-		a.onNext( Math.random() );
+		observer.next( Math.random() );
 	}, 1000 );
 } );
 ```
 
-In this snippet, `a` is the Observable, and unsurprisingly, the separate observer is called `observer`; it's able to "observe" some events (like our `setInterval(..)` loop) and publish events to the `a` observable stream.
+In this snippet, `a` is the Observable, and unsurprisingly, the separate observer is called `observer`; it's able to "observe" some events (like our `setInterval(..)` loop) and ~~publish events to the `a` observable stream~~.
```

Comment:
And we really cannot push data into an Observable. Ever. (Unless it's a Subject, but that's because it's an Observer)